modules/4337/test/erc4337/ERC4337ModuleExisting.spec.ts

```diff
@@ -78,19 +89,24 @@ describe('Safe4337Module - Existing Safe', () => {
       const safeOpHash = calculateSafeOperationHash(await validator.getAddress(), safeOp, await chainId())
       const signature = buildSignatureBytes([await signHash(user1, safeOpHash)])
       const userOp = buildUserOperationFromSafeUserOperation({ safeOp, signature })
-      await entryPoint.executeUserOp(userOp, 0)
-      expect(await ethers.provider.getBalance(await safe.getAddress())).to.be.eq(ethers.parseEther('0.5'))
-      await expect(entryPoint.executeUserOp(userOp, 0)).to.be.revertedWithCustomError(entryPoint, 'InvalidNonce').withArgs(0)
+      await entryPoint.handleOps([userOp], user1.address)
+      expect(await ethers.provider.getBalance(randomAddress)).to.be.eq(ethers.parseEther('0.5'))
+      await expect(entryPoint.handleOps([userOp], user1.address))
+        .to.be.revertedWithCustomError(entryPoint, 'FailedOp')
+        .withArgs(0, 'AA25 invalid account nonce')
     })
 
     it('should execute contract calls with fee', async () => {
       const { user1, safe, validator, entryPoint } = await setupTests()
+      const randomAddress = ethers.Wallet.createRandom().address
+      const randomAddress2 = ethers.Wallet.createRandom().address
```

Comment:
Suggestion: Might make sense to name the fee beneficiary, so the test is a bit more self-explanatory:

```suggestion
      const feeBeneficiary = ethers.Wallet.createRandom().address
      const randomAddress = ethers.Wallet.createRandom().address
```